platform/lib/pipeline/componentReferenceDocument.js

```diff
@@ -156,6 +156,10 @@ class ComponentReferenceDocument extends MarkdownDocument {
     this._frontmatter['bentoPath'] = hasBento;
   }
 
+  set bentoDocUrl(url) {
+    this._frontmatter['bentoDocUrl'] = url;
```

Comment:
_frontmatter is never undefined?

Reply:
it's defined in the parent class platform/lib/pipeline/markdownDocument.js